README.md

```diff
@@ -658,7 +658,7 @@ If you're using the `X-SourceMap` header instead, you can just omit `sourceMap.u
   `Function.prototype.name`. See also: the `keep_fnames` [mangle option](#mangle).
 
 - `keep_infinity` (default: `false`) -- Pass `true` to prevent `Infinity` from
-  being compressed into `1/0`, which may cause performance issues on Chrome.
+  being compressed into `1/0`, which may cause performance issues on old versions of Chrome.
```

Comment:
It would be better to state when Chrome reached performance parity for `1/0` and `Infinity`. If not the specific version, at least the year.